sklearn/model_selection/_validation.py

```diff
@@ -544,9 +544,14 @@ def permutation_test_score(estimator, X, y, labels=None, cv=None,
         The target variable to try to predict in the case of
         supervised learning.
 
-    labels : array-like, with shape (n_samples,), optional
-        Group labels for the samples used while splitting the dataset into
-        train/test set.
+    groups : array-like, with shape (n_samples,), optional
+        Group labels used to constrain the permutation to specific subsets of
+        data.
```

Comment:
This could be improved as suggested by @jnothman in https://github.com/scikit-learn/scikit-learn/pull/6660/files#r77563448